src/css/components/panel.scss

```diff
@@ -29,30 +28,32 @@ $panel-padding: $grid-3;
 
 }
 
-// TODO this is a hack, panel-row borders need to be reconciled, panel-row vs
-// panel-row-bordered
-.panel-row-clean {
-  border: none !important; // stylelint-disable-line declaration-no-important
-}
-
-h1 + .panel,
-h2 + .panel,
-h3 + .panel,
-h4 + .panel {
-  margin-top: 2rem;
+.panel-header {
+  align-items: flex-end;
+  display: flex;
 }
 
 .panel-title {
+  flex: 1 0 0;
   font-size: $sans-s7;
   font-weight: 400;
   line-height: $lh-single;
-  margin-bottom: $grid-3;
+  margin-bottom: $grid-5;
 }
 
 .panel-secondary {
```

Comment:
i.e. the logs message in the following panel:

![screen shot 2017-01-06 at 12 29 01 pm](https://cloud.githubusercontent.com/assets/11464021/21731920/b3e8ef88-d40b-11e6-82d4-8d1f4f8d15be.png)